File: app/src/main/java/org/stepic/droid/core/CertificateComponent.java
package org.stepic.droid.core;


import org.stepic.droid.ui.fragments.CertificateFragment;

import dagger.Subcomponent;

@PerFragment
@Subcomponent(modules = {CertificateModule.class})
public interface CertificateComponent {
    void inject(CertificateFragment certificateFragment);
}

File: app/src/main/java/org/stepic/droid/core/CertificateComponent.kt
package org.stepic.droid.core

import dagger.Subcomponent
import org.stepic.droid.ui.fragments.CertificateFragment

@PerFragment
@Subcomponent(modules = arrayOf(CertificateModule::class))
interface CertificateComponent {
    fun inject(certificateFragment: CertificateFragment)
}
